address;source_location;insn;indent
0x131c60;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:468;MOV	0x18(%RBX),%RAX;
0x131c64;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:467;LEA	0x10(%RBX),%RDI;
0x131c68;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:469;MOV	%R13,%RSI;
0x131c6b;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:469;MOV	%RDI,%RDX;
0x131c6e;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:469;CALLQ	0xb8(%RAX);
0x131c74;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:466;MOV	(%RBX),%RBX;
0x131c77;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:466;CMP	%RBX,%RBP;
0x131c7a;/usr/local/src/conda/python-3.9.19/Modules/gcmodule.c:466;JNE	131c60 <deduce_unreachable+0x60>;
